kruize-clowdapp.yaml

```diff
@@ -157,7 +157,7 @@ parameters:
 - description: Kruize image tag
   name: KRUIZE_IMAGE_TAG
   required: true
-  value: "870ea25"
+  value: "69020a42"
```

Comment:
@khansaad  Thank you for pull-request. Observation - I don't see any image created with this refs number under [cloudservices/autotune](https://quay.io/repository/cloudservices/autotune?tab=tags). Also, it includes just first 7 digits.

Reply:
Updated the digits count to first 7.